### PR TITLE
refactoring: notifier is util now

### PR DESCRIPTION
**Pull request by talyguryn**
notifier is util now

### CODE REVIEW

**Review by neSpecc — CHANGES_REQUESTED** [review]
No description provided.

**Review by gohabereg — APPROVED** [review]

File: src/types-internal/editor-modules.d.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import InlineToolbar from '../components/modules/toolbar/inline';
 import Toolbox from '../components/modules/toolbar/toolbox';
 import BlockSettings from '../components/modules/toolbar/blockSettings';
 import Paste from '../components/modules/paste';
-import Notifier from '../components/modules/notifier';
+import Notifier from '../components/utils/notifier';
```
Comment (neSpecc) [inline src/types-internal/editor-modules.d.ts]: it is not a module anymore, so it should not be here

**Review by neSpecc — CHANGES_REQUESTED** [review]
And what about usages?

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
 - `Fix` — Fix copy in FireFox [1625](https://github.com/codex-team/editor.js/issues/1625)
 - `Refactoring` - The Sanitizer module is util now.
 - `Refactoring` - Tooltip module is util now.
+- `Refactoring` - Notifier module now is a util.
```
Comment (neSpecc) [inline docs/CHANGELOG.md]: wrong section

**Review by neSpecc — APPROVED** [review]

**Pull request merged**